discussment-core/src/main/java/org/danekja/discussment/core/accesscontrol/service/impl/PermissionService.java

```diff
@@ -15,12 +16,12 @@
 import java.util.List;
 import java.util.Objects;
 
-public class PermissionService implements PermissionManagementService, AccessControlService {
+public class PermissionService implements PermissionManagementService, AccessControlService, AccessControlManagerService {
 
-    private NewPermissionDao permissionDao;
+    private PermissionDao permissionDao;
     private DiscussionUserService userService;
 
-    public PermissionService(NewPermissionDao permissionDao, DiscussionUserService userService) {
+    public PermissionService(PermissionDao permissionDao, DiscussionUserService userService) {
         this.permissionDao = permissionDao;
```

Comment:
The original methods (which dont take user parameter) should call the new ones with the current user parameter instead of having own implementation